Clarify Next steps (#202)
redhat-documentation/modular-docs: modular-docs-manual/content/topics/creating_procedure_modules.adoc

Focus on relevant resources that might interest the user. Do not list resources for completeness. If a resource applies to all of the modules in a user story, consider listing the resource in the Additional resources section of the assembly file. If you do this, consider including the resource in the procedure module as a comment.

.Procedure next steps

> This section is optional. You can use it for pointing the reader to instructions covering tasks that might follow after the current procedure. Place this section after the Additional resources section with the *Next steps* heading. Alternatively, if the user should perform the next steps before the verification steps, place the section in between the Procedure section and the Verification steps section.

Reviewer: ```suggestion
This section is optional. You can use it for pointing the reader to tasks that might follow after the current procedure. Place this section after the *Additional resources* section and use the *Next steps* heading. Alternatively, if the user needs to perform the next steps before the verification steps, place the section between the *Procedure* section and the *Verification* section.
```

Reviewer 2: Suggest changing 'pointing' to 'directing'. I really don't think we should add a Next steps section before the verification. If the 'next steps' are needed to complete the procedure then they should be part of the procedure.

Author: > Suggest changing 'pointing' to 'directing'. 

Changed, thank you.

> I really don't think we should add a Next steps section before the verification. If the 'next steps' are needed to complete the procedure then they should be part of the procedure.

However, an example of why we should explicitly allow such order was one of the reasons for filing this issue. Marc Muehlfeld and I believe that in cases such as [1] is this the best order.

[1] https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/securing_networks/creating-and-managing-tls-keys-and-certificates_securing-networks#creating-a-private-ca-using-openssl_creating-and-managing-tls-keys-and-certificates
